doc/nrf/glossary.rst

```diff
@@ -414,21 +457,27 @@ Glossary
    Receive Data (RXD)
       A signal line in a serial interface that receives data from another device.
 
+   Received Signal Strength Indication (RSSI)
+      An indication of the power of a received radio signal.
+
    Repository
       The entire set of files and folders of which a project is composed, together with the revision history of each file.
       It is often composed of multiple branches.
       It is also known as *Git repository* or *Git project*, and is similar to a *Git submodule*.
 
+   Request to Send (RTS)
+      In :term:`flow control <Hardware Flow Control (HWFC)>`, the transmitting end is ready and requesting the far end for permission to transfer data.
+
    Root of Trust (RoT)
```

Comment:
Maybe we need to add RPC (remote procedure call)

Reply:
@peknis Do you also have a clear definition for it you would like to add?

Reply:
Did not find in our docs, but Google gave for example this: https://www.w3.org/History/1992/nfs_dxcern_mirror/rpc/doc/Introduction/WhatIs.html

Reply:
added, will tag reviewers on that one